browser/component_updater/brave_component_updater_configurator.cc

```diff
@@ -185,11 +185,7 @@ BraveConfigurator::GetProtocolHandlerFactory() const {
 
 update_client::RecoveryCRXElevator BraveConfigurator::GetRecoveryCRXElevator()
     const {
-#if defined(GOOGLE_CHROME_BUILD) && defined(OS_WIN)
```

Comment:
why is this removed?

Reply:
it seems like this was wrong from the start and should have be `IS_OFFICIAL_BUILD` for us?

Reply:
GOOGLE_CHROME_BUILD was never define for this file, as far as I can see, so the code inside was never used.

Reply:
@bbondy is going to look into this for a potential follow-up